src/app/services/api/api.service.ts

```diff
@@ -204,6 +204,17 @@ export class ApiService {
             .toPromise();
     }
 
+    /** Fetch/query BNS domain. */
+    async fetchBNSDomain(domain_name: string, tld: string): Promise<any> {
+        await this._hasPingedApi();
+        return this._http
+            .post<any>(`${this.httpApi}/v1/account/bns`, {
```

Comment:
Could we get some stronger typings here?  The api is versioned so we should know what schema to expect.